src/btree/bt_read.c

```diff
@@ -536,26 +541,23 @@ __page_read(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
 	case WT_REF_LIMBO:
 	case WT_REF_LOOKASIDE:
 		WT_ERR(__page_read_lookaside(
-		    session, ref, previous_state, &final_state));
+		    session, ref, previous_state, &final_state, &prepared));
 		break;
 	}
 
 	/*
 	 * Once the page is instantiated, we no longer need the history in
 	 * lookaside.  We leave the lookaside sweep thread to do most cleanup,
-	 * but it can only remove keys that skew newest (if there are entries
-	 * in the lookaside newer than the page, they need to be read back into
-	 * cache or they will be lost).
-	 *
-	 * There is no reason for the lookaside remove should fail, but ignore
-	 * it if for some reason it fails, we've got a valid page.
+	 * but it can only remove committed updates and keys that skew newest
```

Comment:
Enhance the comment to indicate that we will remove if the page has prepared updates.